### ISSUE TITLE
Netlify hosting only allows one deployment maintainer

### ISSUE THREAD

**Issue by almereyda** [La-MYNE/Website#7]
The free tier of Netlify, where this repository is hosted, only allows for one maintainer. This is currently @unteem as of my knowledge.

Unfortunately this comes with a high [bus factor](https://en.wikipedia.org/wiki/Bus_factor) and is an instittutional enclosure by a freemium marketing campaign, leading to a [single point of failure](https://en.wikipedia.org/wiki/Single_point_of_failure).

Rethinking hosting options and organisational share of responsibility may be useful here.

**Comment by unteem** [comment]
true. I need to move it to IndieHosters servers, that would solve that issue
